src/test/java/cz/cvut/kbss/termit/rest/WorkspaceControllerTest.java

```diff
@@ -44,14 +45,14 @@
 
     @BeforeEach
     void setUp() {
-        MockitoAnnotations.initMocks(this);
+        MockitoAnnotations.openMocks(this);
```

Comment:
Should not they be closed @AfterEach ?

Reply:
No, Mockito API replaced `initMocks` with `openMocks`, but there is no explicit close.